inf-clojure.el

```diff
@@ -726,7 +726,10 @@ If you are using REPL types, it will pickup the most approapriate
 (define-obsolete-variable-alias 'inf-clojure-completion-command 'inf-clojure-completion-form "2.0.0")
 
 (defcustom inf-clojure-completion-form-lumo
-  "(doall (map str (lumo.repl/get-completions \"%s\")))"
+  "(let [ret (atom)]
```

Comment:
I'll do that and squash. But isn't this a bit mean to someone who wants to prettyprint the results when emacs is always setting it to false?